src/cljs/rems/actions/request_decision.cljs

```diff
@@ -11,26 +10,32 @@
 (rf/reg-fx
  ::fetch-potential-deciders
  (fn [[user on-success]]
-  (fetch (str "/api/applications/deciders")
-         {:handler on-success
-          :headers {"x-rems-user-id" (:eppn user)}})))
+   (fetch (str "/api/applications/deciders")
+          {:handler on-success
+           :headers {"x-rems-user-id" (:eppn user)}})))
 
 (rf/reg-event-fx
  ::open-form
  (fn [{:keys [db]} _]
-  {:db (assoc db
-              ::comment ""
-              ::potential-deciders #{}
-              ::selected-deciders #{})
-   ::fetch-potential-deciders [(get-in db [:identity :user])
-                               #(rf/dispatch [::set-potential-deciders %])]}))
+   {:db (assoc db
+               ::comment ""
+               ::potential-deciders #{}
+               ::selected-deciders #{})
+    ::fetch-potential-deciders [(get-in db [:identity :user])
+                                #(rf/dispatch [::set-potential-deciders %])]}))
 
 (rf/reg-sub ::potential-deciders (fn [db _] (::potential-deciders db)))
+(rf/reg-event-db
+ ::set-potential-deciders
```

Comment:
was this event really missing previously?

Reply:
It was accidentally removed in commit 6cac8dd0c3de8e2da0860674f7c38c4492df111e